doc/examples/applications/plot_rank_filters.py

```diff
@@ -226,11 +235,12 @@
 # The following example shows how local auto-level enhances the camara man
 # picture.
 
-from skimage.filters.rank import autolevel
```

Comment:
Here and in other places, you typically want to remove a blank line as well (either the one above or the one below) in addition to the import line. Otherwise you end up with an extra blank line.

Reply:
Thank you, I will fix it. I contributed to an another open-source project once and it used pre-commit and before committing, it automatically edited out all the style issues. I want to know if there exists any tool like that for Scikit-image?